src/main/kotlin/io/github/gradlenexus/publishplugin/InitializeNexusStagingRepository.kt

```diff
@@ -40,7 +39,6 @@ constructor(
     private val stagingRepositoryId: (String) -> Unit
 ) : AbstractNexusStagingRepositoryTask(objects, extension, repository) {
 
-    @Optional
```

Comment:
It's only required if `stagingProfileId` is not set. Thus, I think `@Optional` was correct, wasn't it?

Reply:
Well, they both can be not set and then there will be `val packageGroup = packageGroup.get()` called. However, it would be problematic to add two-field present validation, so probably `packageGroup.get()` is good enough, especially as it reports the name of the not provided property. I will restore `@Optional`.